packages/api/utils/response.ts

```diff
@@ -186,3 +187,27 @@ export class ValidationErrorResponse extends ErrorResponse {
     super(error, ErrorStatusCode.BAD_REQUEST, headers, multiValueHeaders);
   }
 }
+export class DatabaseError {
+  public readonly errorResponse: ErrorResponse;
+  constructor(errorResponse: ErrorResponse) {
+    this.errorResponse = errorResponse;
+  }
+}
+
+export type DatabaseResult = DatabaseError | UpdateCommandOutput; // | GetCommandOutput | DeleteCommandOutput;
```

Comment:
Why are the last two commented out? Does it cause an error? Let's just add them or delete them as variants of the type.